search/searcher/search_phrase.go

```diff
@@ -169,7 +169,12 @@ func (s *PhraseSearcher) checkCurrMustMatch(ctx *search.SearchContext) *search.D
 // satisfied, and these locations are returned.  otherwise 0 and either
 // a nil or empty TermLocationMap
 func (s *PhraseSearcher) checkCurrMustMatchField(ctx *search.SearchContext, tlm search.TermLocationMap) (int, search.TermLocationMap) {
-	paths := findPhrasePaths(0, nil, s.terms, tlm, nil, 0)
+	// temporarily turn flat terms []sting into [][]string
+	terms := make([][]string, len(s.terms))
```

Comment:
this is temporary and goes away in the next commit, i just broke it out so it was easier to follow